src/Nethermind/Nethermind.Crypto/BlsSigner.cs

```diff
@@ -8,13 +8,15 @@
 
 namespace Nethermind.Crypto;
 
+using G1 = Bls.P1;
 using G1Affine = Bls.P1Affine;
 using G2 = Bls.P2;
 using G2Affine = Bls.P2Affine;
 using GT = Bls.PT;
 
 public static class BlsSigner
```

Comment:
maybe a link to BLS RFC or similar would be good for people who are not experts in Bls, like me 😄